test/spec.model.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const SessionModel = require('../lib/model');
+
+describe('session model', () => {
+
+    it('exports a function', () => {
+        SessionModel.should.be.a.function;
+    });
+
+    describe('constructor', () => {
+        it('should throw an error if no options are provided', () => {
+            expect(() => new SessionModel()).to.throw();
+        });
+
+        it('should throw an error if no key is provided in options', () => {
+            expect(() => new SessionModel(null, { session: {} })).to.throw();
+        });
+
+        it('should throw an error if no session is provided in options', () => {
+            expect(() => new SessionModel(null, { key: {} })).to.throw();
```

Comment:
I've added in some explicit testing and throwing, corrected the test case for session and added in the throw messages.
:+1: